certify-service/src/main/resources/application-local.properties

```diff
@@ -186,33 +186,30 @@ mosip.certify.key-values={\
                                   'text_color': '#7C4616'}},\
                     'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
                  },\
-                  'LifeInsuranceCredential':{\
-                      'format': 'ldp_vc',\
-                      'scope' : 'life_insurance_vc_ldp',\
-                      'cryptographic_binding_methods_supported': {'did:jwk'},\
-                      'credential_signing_alg_values_supported': {'Ed25519Signature2020'},\
-                      'proof_types_supported': {'jwt': {'proof_signing_alg_values_supported': {'RS256', 'ES256'}}},\
-                      'credential_definition': {\
-                          'type': {'VerifiableCredential', 'LifeInsuranceCredential'},\
-                          'credentialSubject': {\
-                              'fullName': {'display': {{'name': 'Name','locale': 'en'}}}, \
-                              'mobile': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
-                              'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
-                              'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
-                              'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
-                              'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
-                              'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
-                              'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
-                              'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
-                              'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                  "DrivingLicenseCredential":{\
+                      'format': 'mso_mdoc',\
```

Comment:
Create new config for DrivingLicenseCredential

Reply:
Adding new config arises `SpEL expression is too long` issue, to avoid this issue, , vd11 support has been removed from application-local.properties temporarily